aiosmtpd/tests/test_handlers.py

```diff
@@ -5,7 +5,7 @@
 
 from aiosmtpd.controller import Controller
 from aiosmtpd.handlers import (
-    AsyncMessage, Debugging, Mailbox, Message, Proxy, Sink)
+    AsyncMessage, Debugging, Mailbox, Message, NEWLINE, Proxy, Sink)
```

Comment:
I know this is a very minor point, but I'm not sure about importing `NEWLINE` here.  The reason is that `NEWLINE` isn't part of the public API for `aiosmtpd.handlers` and maybe we should only import public names in the test?  OTOH, maybe that's a dumb restriction. ;) 

It would be simple to re-define in `test_handlers.py`, but maybe it's not worth it.  What do you think?